contrib/yolov5/tests/test_torchhub.py

```diff
@@ -0,0 +1,42 @@
+#  Copyright (c) 2021 Rikai Authors
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from pathlib import Path
+
+from pyspark.sql import SparkSession
+
+from rikai.contrib.yolov5.transforms import OUTPUT_SCHEMA
+
+
+def test_yolov5(spark: SparkSession):
+    work_dir = Path().absolute().parent.parent
+    image_path = f"{work_dir}/python/tests/assets/test_image.jpg"
+    spark.sql(
+        f"""
+CREATE MODEL yolov5
+FLAVOR yolov5
```

Comment:
Sorry I just catch this late in the game. So flavor here is `yolov5`, it means a model-special runner?

cc @changhiskhan is this desired behavior?

Reply:
See https://github.com/eto-ai/rikai/pull/376 and https://github.com/eto-ai/rikai/blob/master/contrib/yolov5/REAMDE.md

Reply:
yeah, yolov5 has a bit of a weird interface so we're using a custom flavor here. @da-tubi a lot of people will look to the tests for examples so maybe let's add another one using the standard pytorch flavor? And in this test just comment why we need the custom flavor.